frontend-ssr/components/custom_menu/Menu.module.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../styles/variables';
+.titleMenu {
+    width: 100%;
+    margin-top: 1em;
+    button {
+        display: flex;
+        background: none;
+        border: none;
+        outline: none;
+        color: color(black);
+        font-weight: bolder;
+        cursor: pointer;
+        padding-left: 2.5em;
+        min-height: 60px;
+        position: relative;
+        &::before {
```

Comment:
I would expect `before & after` to be reversed. The `line` to be `after` since it comes after the button, and the Dot to be `before`.